presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -225,10 +225,11 @@ Property Name                                           Description
 
 ``iceberg.enable-parquet-dereference-pushdown``         Enable parquet dereference pushdown.                          ``true``
 
-``iceberg.hive-statistics-merge-strategy``              Determines how to merge statistics that are stored in the     ``NONE``
-                                                        Hive Metastore. The available values are ``NONE``,
-                                                        ``USE_NULLS_FRACTION_AND_NDV``, ``USE_NULLS_FRACTIONS``
-                                                        and, ``USE_NDV``
+``iceberg.hive-statistics-merge-strategy``              Comma separated list of statistics to use from the
+                                                        hive metastore to override Iceberg table statistics. Only
```

Comment:
```suggestion
                                                        Hive Metastore to override Iceberg table statistics.
```
The sentence "Only valid when the Iceberg connector is configured with Hive." could be moved out of the middle of the paragraph to highlight it, which improves readability by making it easier for the user to quickly read, and dismiss this property if it's irrelevant to their situation. (I'm including a screenshot only because I'd already mocked this up in my local build to see how it looks.)
<img width="656" alt="Screenshot 2024-03-26 at 1 08 17 PM" src="https://github.com/prestodb/presto/assets/7013443/6867aec6-de50-4ef3-909f-8df467d498fe">